java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -13,6 +13,15 @@
  */
 public interface StringCommands {
 
+    /**
+     * Removes the specified <code>keys</code>. A key is ignored if it does not exist.
+     *
+     * @see <a href="https://redis.io/commands/del/">redis.io</a> for details.
+     * @param keys the keys we wanted to remove.
+     * @returns the number of keys that were removed.
```

Comment:
```suggestion
     * Remove the specified <code>keys</code>. A key is ignored if it does not exist.
     *
     * @see <a href="https://redis.io/commands/del/">redis.io</a> for details.
     * @param keys The keys to remove.
     * @returns The number of keys that were removed.
```

Reply:
@return and not returns? @Yury-Fridlyand the Removes are from their code are you sure you want to change that?

Reply:
Oh right, please use `@return` in java, not `@returns`

Reply:
Changing documentation should be a separate PR and we should update all clients. 
This is from the Node documentation: https://github.com/aws/glide-for-redis/blob/main/node/src/BaseClient.ts#L418